tests/iomethods_module_test.py

```diff
@@ -0,0 +1,123 @@
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                                [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]]]
+
+        expectedTrimmedData = ma.array(expectedTrimmedData)
+
+        expectedTimes = [datetime(2009, 8, 31, 0, 0)]
+        expectedLats = [9.875, 10.125, 10.375, 10.625, 10.875, 11.125, 11.375, 11.625, 11.875, 12.125, 12.375,\
+                        12.625, 12.875, 13.125, 13.375, 13.625, 13.875, 14.125, 14.375, 14.625, 14.875]
+        expectedLons = [9.875, 10.125, 10.375, 10.625, 10.875, 11.125, 11.375, 11.625, 11.875, 12.125, 12.375,\
+                        12.625, 12.875, 13.125, 13.375, 13.625, 13.875, 14.125, 14.375, 14.625, 14.875]
+
+        self.assertTrue(ma.allclose(expectedTrimmedData, trimmedData), "Trimmed and expected data differ!")
+
+        self.assertEqual(expectedTimes, times, "Returned time and expected time differ!")
+
+        latsListTest = (expectedLats == trimmedLats)
+        latsListTest = latsListTest if type(latsListTest) is not np.ndarray else latsListTest.all()
+        self.assertTrue(latsListTest, "Trimmed and expected Lat ranges differ!")
+
+        lonsListTest = (expectedLons == trimmedLons)
+        lonsListTest = lonsListTest if type(lonsListTest) is not np.ndarray else lonsListTest.all()
+        self.assertTrue(lonsListTest, "Trimmed and expected Lon ranges differ!")
+
+    def test_read_merg_and_write_merg(self):
+        '''
+            Purpose:: Tests the functions '_read_merg_file' and '_write_MERG_pixel_to_ncdf' from iomethods.py together
+        '''
+        FILE = "../datadir/MERG/merg_2006091100_4km-pixel"
```

Comment:
Please be mindful of variable names. FILE is not a global or constant variable and should not be capitalized.